hivemind/compression/quantization.py

```diff
@@ -132,6 +130,9 @@ class BlockwiseQuantization(Quantization):
     def quantize(
         self, tensor: torch.Tensor, allow_inplace: bool = False
     ) -> Tuple[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
+        # This runs actual import only on the 1st call, copies references after that
+        from bitsandbytes.functional import quantize_blockwise
```

Comment:
Since this is no longer under a conditional import (and will thus cause ImportError instead of NameError), can we wrap this statement instead of the `quantize_blockwise` call below with try-except? The same applies to `extract`